### PR TITLE
Set lower default verbosity and add config options.

### PR DESCRIPTION
**Pull request by Carreau**
I think this should cover most of the cases.

See #149

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -63,3 +63,24 @@ You can disable this behavior by setting the following config:
...
+ - `jupyterlite_debug`, boolean (`False`|`True`), passes the `--debug` flag  to
+   `jupyterlite build`
+
+It also has the following configuration options:
+
+ - `jupyterlite_log_level`, a `str`, default to `"WARN"` or `None`
+ - `jupyterlite_verbosity` a `str`, default to `"0"` or `None`
+ - `jupyterlite_reporter` a `str`, default to `"zero"` or `None`
+
+Jupyterlite-sphinx decreases verbosity by deafult, these can be set to `None` to restore the jupyterlite default.
```
Comment (steppi) [inline docs/configuration.md:83]: I think we should mention verbosity controls the verbosity setting of `doit` which is used in `jupyterlite`, and link to the `doit` config docs, https://smarie.github.io/python-doit-api/api_reference/.

And also mention that `jupyterlite_log_level` controls the `log_level` for `jupyterlite build` itself. I can't find `--log-level` in the `jupyterlite` docs though.

It's also unclear to me what "default to `"Warn"` or `"None"`" etc. means
Maybe just "defaults to `"Warn"`` etc.? And then just mention that setting these values to `None` gives the defaults?

**Review by steppi — APPROVED** [review]
Looks great! Thanks @Carreau. Just some minor comments on the documentation.

File: docs/configuration.md (hunk 1 above)
Comment (steppi) [reply docs/configuration.md:83]: oh, I get it now. i think I'd understand if it was something like

a `str` (defaults to `"WARN"`) or `None`

File: docs/configuration.md [hunk 2]
```diff
@@ -63,3 +63,24 @@ You can disable this behavior by setting the following config:
...
+
+See `jupyterlite build` documentation for the command line arguments of the same
+name.
```
Comment (steppi) [inline docs/configuration.md:86]:
```suggestion
See the `jupyterlite build` documentation for info on `log-level`. `verbosity` and `reporter` control the configuration
of [doit](https://smarie.github.io/python-doit-api/api_reference/), which is used internally in `jupyterlite`.
```

**Comment by steppi** [comment]
I fixed the docs and merged it since I want to get this in ASAP since its causing serious trouble in SciPy. I plan to make a release now.

**Comment by matthewfeickert** [comment]
This resolves Issue https://github.com/jupyterlite/jupyterlite-sphinx/issues/106 as this changes the expected type from `PosixPath` to `str`.

**Comment by matthewfeickert** [comment]
This also breaks builds as there is now no longer a `_build/html/lite` directory.

File: jupyterlite_sphinx/jupyterlite_sphinx.py [hunk 3]
```diff
@@ -574,21 +580,36 @@ def jupyterlite_build(app: Sphinx, error):
...
-            "--output-dir",
-            os.path.join(app.outdir, JUPYTERLITE_DIR),
-            *apps_option,
-            "--lite-dir",
-            jupyterlite_dir,
+            c
+            for c in (
+                "jupyter",
+                "lite",
+                "doit" "build",
```
Comment (matthewfeickert) [inline jupyterlite_sphinx/jupyterlite_sphinx.py:587]: @Carreau I think you missed a comma here(?).

I don't have time now, but I will try to debug this more tonight US time.

**Comment by matthewfeickert** [comment]
Thanks @steppi. As a [`jupyterlite-sphinx` `v0.13.0`](https://github.com/jupyterlite/jupyterlite-sphinx/releases/tag/v0.13.0) contains bugs that are breaking builds can you please yank it from PyPI and update the release notes to reflect this?

File: jupyterlite_sphinx/jupyterlite_sphinx.py [hunk 4]
```diff
@@ -574,21 +580,36 @@ def jupyterlite_build(app: Sphinx, error):
...
+                "--contents",
+                os.path.join(app.srcdir, CONTENT_DIR),
+                "--output-dir",
+                os.path.join(app.outdir, JUPYTERLITE_DIR),
+                *apps_option,
+                "--lite-dir",
+                jupyterlite_dir,
+                "--log-level" if jupyterlite_log_level is not None else None,
+                jupyterlite_log_level,
+                "--",
```
Comment (matthewfeickert) [inline jupyterlite_sphinx/jupyterlite_sphinx.py:600]: I think this `--` is also an unintended addition. ?

Comment (matthewfeickert) [reply jupyterlite_sphinx/jupyterlite_sphinx.py:600]: c.f. https://github.com/jupyterlite/jupyterlite-sphinx/pull/152

Comment (steppi) [reply jupyterlite_sphinx/jupyterlite_sphinx.py:600]: That's needed to pass the args to `doit` in `jupyterlite build`. See https://github.com/jupyterlite/jupyterlite/issues/1351

**Comment by steppi** [comment]
> Thanks @steppi. As [`jupyterlite-sphinx` `v0.13.0`](https://github.com/jupyterlite/jupyterlite-sphinx/releases/tag/v0.13.0) contains bugs that are breaking builds can you please yank it from PyPI and update the release notes to reflect this?

Yes. That's definitely needed. I think one of @martinRenou or @jtpio will need to yank it though.

**Comment by jtpio** [comment]
Just yanked the `0.13.0` release:

image

FYI I added @Carreau to the project on PyPI. @steppi what is your handle on PyPI, so I can also add you there?

**Comment by Carreau** [comment]
Sorry I should have tested more. I mostly checked it was not failing at the end of yesterday. Thanks for caching that,.

**Pull request merged**